chart/values.yaml

```diff
@@ -261,6 +261,8 @@ duckDBIndex:
   urlTemplate: "/datasets/%s/resolve/%s/%s"
   # the maximum size of the split parquets.
   maxParquetSizeBytes: "100_000_000"
+  # the time interval at which a downloaded index will be considered as expired and will be deleted 
+  expiredTimeIntervalSeconds: 600
```

Comment:
On first thought, it could be a lot more, like 1 day, or 3 days. We don't really have limits on the storage, and it would reduce the latency for datasets that are searched often.

Reply:
Sure, I changed it to 3 days for prod https://github.com/huggingface/datasets-server/pull/1536/commits/f38ba31e3b7664ee3322416f71f07444cfddf906